### ISSUE TITLE
VQGAN Problem 

### ISSUE THREAD

**Issue by twoDragonBear** [xiyuez2/MapPrior#1]
hi, thanks for your great work.
I want to know how many epochs for the stage 1 training, i can't find the config in the code.

**Comment by xiyuez2** [comment]
I trained it until the validation loss converge. It took ~40 epochs.
